### DISCUSSION TITLE
蓝牙/wifi/dtb问题

### DISCUSSION THREAD

**Discussion by A-nemon-e** [ophub/amlogic-s9xxx-armbian#1704] [Amlogic]
问个很弱智问题，手上两个盒子，用的相同dtb（meson-gxl-s905l2-x7-5g）和overload（s905x-s912），一个mt7661rsn，一个mt7668rsn，用一样的内核，为什么7661的蓝牙能用而7668的不行？（我看到您的armbian自带了mtk的sdio的bt驱动）
（一些其他信息是，在未额外编译驱动的情况下，7661的wifi在nmcli中被显示，7668的wifi就干脆没有显示；反编译的dts干脆就没找到蓝牙的字眼，那个7661的蓝牙又为什么能用呢）
一共有哪些因素影响了wifi和bt的使用？dtb不影响这些设备的识别吗？u-boot参与硬件识别吗？和dtb的关系是什么呢？

**Comment by ophub** [comment]
mt7668 wifi和蓝牙测试内核及使用方法： https://github.com/ophub/amlogic-s9xxx-armbian/issues/3610#issuecomment-5091652991
